.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -14,5 +14,5 @@ Note that you need to sign Google's CLA to contribute.
 If applicable:
 - [ ] I have updated `CHANGELOG.md`
 - [ ] I have updated the documentation (README.md, docs/, demos/)
-- [ ] I have updated the config schema (src/config-schema.json)
+- [ ] I have updated the config schema (cli/src/config-schema.json)
```

Comment:
nit: Could you update the commit message to be a bit more specific? Someone reading it should be able to tell if they need to look at the reset of the commit, and "Fix paths" sounds like it might be a bug fix. Something like "docs: fix missed paths from src->cli/src move" might be enough.